src/Command/CommandBase.php

```diff
@@ -407,9 +407,10 @@ public function run(InputInterface $input, OutputInterface $output): int {
    * Add argument and usage examples for applicationUuid.
    */
   protected function acceptApplicationUuid() {
-    $this->addArgument('applicationUuid', InputArgument::OPTIONAL, 'The Cloud Platform application UUID or alias')
+    $this->addArgument('applicationUuid', InputArgument::OPTIONAL, 'The Cloud Platform application UUID or alias (i.e. a sitegroup optionally prefixed with the realm)')
```

Comment:
we should not use the term "sitegroup". That is an internal only machine name that we have been trying to move away from for a few years.

Reply:
> (i.e. an application name optionally prefixed by the realm.)